package.json

```diff
@@ -26,6 +27,7 @@
   },
   "homepage": "https://github.com/dwyl/terminate",
   "dependencies": {
+    "goodparts": "1.2.1",
```

Comment:
`goodparts` should be in `devDependendencies` as `terminate` does not depend on it for any _core_ functionality.